_partials/integrations/hibernate-ogm.html.haml

```diff
@@ -0,0 +1,3 @@
+The Hibernate team maintains the
```

Comment:
I have published this branch on the staging site so you can look at it. 
http://stg-ispn.rhcloud.com/integrations/